cmd/crc/cmd/version.go

```diff
@@ -0,0 +1,25 @@
+package cmd
+
+import (
+	"fmt"
+
+	crcPkg "github.com/code-ready/crc/pkg/crc"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(versionCmd)
+}
+
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Get CodeReady Container version.",
+	Long:  "Get the CodeReady Container version currently installed.",
+	Run: func(cmd *cobra.Command, args []string) {
+		runPrintVersion(args)
+	},
+}
+
+func runPrintVersion(arguments []string) {
+	fmt.Printf(fmt.Sprintf("version: %s+%s\n", crcPkg.GetCRCVersion(), crcPkg.GetCommitSha()))
```

Comment:
@LalatenduMohanty why you need to use `Sprintf` here since your are not storing the string but just print it out

```golang
fmt.Printf("version: %s+%s\n", crcPkg.GetCRCVersion(), crcPkg.GetCommitSha())
```

Reply:
@praveenkumar good catch, Initially I was doing some string manipulation, but at the end it was not required, but I forgot to remove it